src/main/kotlin/com/expedia/graphql/schema/generator/directive/DirectiveWiringHelper.kt

```diff
@@ -25,6 +25,9 @@ import graphql.schema.idl.WiringFactory
  */
 class DirectiveWiringHelper(private val wiringFactory: WiringFactory, private val manualWiring: Map<String, SchemaDirectiveWiring> = mutableMapOf()) {
 
+    /**
+     * Wire up the directive based on the GraphQL type
+     */
```

Comment:
Linting warning about undocumented public functions

Reply:
I think that lint warning should be disabled.

Reply:
I think we should keep this one. As library creators we should document all the points that our users can interact with.

It is only a warning right now and does not fail.